website/server/controllers/api-v3/tasks/groups.js

```diff
@@ -13,9 +14,18 @@ import {
   getTasks,
   moveTask,
 } from '../../../libs/taskManager';
+import apiMessages from '../../../libs/apiMessages';
 
 let requiredGroupFields = '_id leader tasksOrder name';
 let types = Tasks.tasksTypes.map(type => `${type}s`);
+
+function canNotEditTasks (group, user, assignedUserId) {
```

Comment:
this function (just statements outside of ay function) seems also used in  website/server/controllers/api-v3/tasks.js, can we abstract it somewhere and add some unit tests?